tests/microbenchmarks.py

```diff
@@ -113,6 +118,7 @@ def foo():
             foo()
     """), timed=True)
 
+
 def test_code(test_case):
     print("Running", "test_code")
     test_case.runAsJava(adjust("""
```

Comment:
At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils